api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -127,19 +127,26 @@ public void setUserService(UserService userService) {
   }
 
   @Override
-  public ResponseEntity<ClusterListResponse> listClusters() {
+  public ResponseEntity<ClusterListResponse> listClusters(String projectName) {
```

Comment:
Naming nit: we should be as precise & consistent with naming as possible here, since the "billing project" concept is now spread across AoU, Terra, and GCP. It may not be reasonable to do too much in this PR, but I think it's an important long-term goal for us to be consistent.

What we're passing through here is an identifier for the GCP project, which Terra generally calls the billing project. GCP calls this the "project ID":

<img width="219" alt="Screen Shot 2019-05-10 at 10 55 19 AM" src="https://user-images.githubusercontent.com/51842/57536559-6a356300-7312-11e9-99ad-a4606173114c.png">

I'm not sure whether Terra refers to the project ID as the "billing project name" or the "billing project ID". But "projectName" is probably not right. This is a globally unique ID that's managed by Google, so we should give it a name that signals that. (And if we someday want to show the separate human-editable GCP project name, then we won't run into mass confusion.)

A few suggested options:
 - cloudProjectId
 - projectId
 - billingProjectId
 - something else?

Reply:
+1 to reaching consistency. projectName is at least partially consistent though - here are the various names for this thing:

- Firecloud: projectName - https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/resources/swagger/api-docs.yaml#L2055
- Leo: googleProject - https://github.com/all-of-us/workbench/blob/48cab8e66c723be603c9931527c82aa39e0dba16/api/src/main/resources/notebooks.yaml#L122
- Firecloud again: arguably a different concept which happens to be the same value "workspaceNamespace" - https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/resources/swagger/api-docs.yaml#L2473
- Our code: mix of projectId, workspaceNamespace, projectName, maybe others

One thing that is critical here would be to understand whether the "projectName" is documented by Firecloud as being the same thing as a Google project ID - or if that's just an implementation detail.

Reply:
I like `billingProjectId` here

Reply:
👍 this has been a source of confusion for me. I think it would be good to take some time and decide on how we want to name things

Reply:
Plugging the relevant tech debt ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-1806

Reply:
Heh.  This sounds awfully familiar to me -> https://broadinstitute.atlassian.net/browse/GAWB-41

For those without access, that's a FireCloud tech debt ticket I filed in 2015 which we never acted on:

"I learned that a Billing Project is the same as a Workspace Namespace.  I knew that both represented a Google Project, but it was not clear to me that they were identical.  Additionally, we have the concept of a Project ID in GoogleServicesDAO, which is the same value yet again.

We should standardize on a single term to avoid confusion, and do a mass renaming of all the others in Rawls."